tests/foreman/cli/test_location.py

```diff
@@ -45,8 +35,10 @@ def test_create_location_with_different_names_positive(self, name):
         @Assert: Location is created successfully and has proper name
 
         """
-        loc = make_location({'name': name})
-        self.assertEqual(loc['name'], name)
+        for name in valid_data_list():
```

Comment:
I have three open PRs with helper module changed at different commits.  To avoid merge conflicts, lets merge this PR, I will submit a patch to handle this issue once this PR is merged.